SomosMasApp/SomosMasApp/App/Shared/Models/SliderData.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SliderData.swift
+//  SomosMasApp
+//
+//  Created by Braian Theiler on 29/03/2022.
+//
+
+import Foundation
+import UIKit
+
+struct SliderData {
+    let title: String?
+    let description: String?
+    let image: UIImage?
+}
+
+let sliderData = [ SliderData(title: "Marcelo Aguirre", description: "Presidente", image: UIImage(named:"Image_1")),
```

Comment:
why the data s defined here??? should be owned by the view controller or the ViewModel or someone. I don{t care you have a barcoded array but not use it directly from  here sign it or something